scripts/devenv.ps1

```diff
@@ -1,7 +0,0 @@
-$pwd = (Get-Location).Path
-
-docker build --pull -t aks-engine .
```

Comment:
I think some people might still be using this one (even though `make dev` is equivalent)

Reply:
Ah right you are. I'll restore this one.